README.md

```diff
@@ -536,23 +553,28 @@ In this section we’ll lay out the differences, benefits, and drawbacks of comm
 granted equity.
 We explain stock options and restricted stock units (RSUs), which the most common forms of
 equity compensation.
-RSUs are more common for larger companies and options more common for startups.
+If you want to jump ahead, RSUs are more common for larger companies and options more
+common for startups.
 
 ### Stock options
 
-🄳 [**Stock options**](https://en.wikipedia.org/wiki/Employee_stock_option) (“employee stock
-options,” in this case) are contracts that allow you to buy shares in the company you work
-for. When you buy shares, you are **exercising** the options.
-Options are not the same as stock;
-they are only the right to buy stock upon and subject to the conditions specified in the
-option agreement.
+🄳 [**Employee stock options**](https://en.wikipedia.org/wiki/Employee_stock_option) (**ESOs**) are
+contracts that allow you to buy shares in the company you work for.
+When you buy shares, you are **exercising** the options.
+
+- 🔸 Options are not the same as stock;
+  they are only the right to buy stock upon and subject to the conditions specified in the
+  option agreement.
 
-🚧 Add a tip about how this isn’t at all the same as trading in options on stocks.
+🚧 Add a tip about how this isn’t at all the same as trading in options on stocks, or
+buying stock options (differentiate ESOs as a type of stock option)
 
-Stock options allow you to buy shares at a *fixed* price per share, the **strike price**.
+🄳 The **strike price** is the *fixed* price per share that stock options allow.
```

Comment:
Awkward phrasing. Alternate:

The **strike price** is the *fixed* price per share at which stock options can be exercised, as set in an employee's stock option agreement.